<!DOCTYPE html>
<html><head><title>AuthenticationFailedException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">7 references to AuthenticationFailedException</div><div class="rA">Azure.Identity (7)</div><div class="rG" id="Azure.Identity"><div class="rF"><div class="rN">AppServiceManagedIdentitySource.cs (1)</div>
<a href="../AppServiceManagedIdentitySource.cs.html#39"><b>39</b>throw new <i>AuthenticationFailedException</i>(MsiEndpointInvalidUriError, ex);</a>
</div>
<div class="rF"><div class="rN">AuthenticationFailedException.cs (1)</div>
<a href="../AuthenticationFailedException.cs.html#23"><b>23</b>: <i>this</i>(message, null)</a>
</div>
<div class="rF"><div class="rN">CloudShellManagedIdentitySource.cs (1)</div>
<a href="../CloudShellManagedIdentitySource.cs.html#36"><b>36</b>throw new <i>AuthenticationFailedException</i>(MsiEndpointInvalidUriError, ex);</a>
</div>
<div class="rF"><div class="rN">CredentialDiagnosticScope.cs (1)</div>
<a href="../CredentialDiagnosticScope.cs.html#79"><b>79</b>exception = new <i>AuthenticationFailedException</i>(exceptionMessage, exception);</a>
</div>
<div class="rF"><div class="rN">Credentials\ChainedTokenCredential.cs (1)</div>
<a href="../Credentials/ChainedTokenCredential.cs.html#111"><b>111</b>throw new <i>AuthenticationFailedException</i>(AuthenticationFailedErrorMessage + e.Message, e);</a>
</div>
<div class="rF"><div class="rN">Credentials\DefaultAzureCredential.cs (1)</div>
<a href="../Credentials/DefaultAzureCredential.cs.html#159"><b>159</b>throw new <i>AuthenticationFailedException</i>(UnhandledExceptionMessage, e);</a>
</div>
<div class="rF"><div class="rN">CredentialUnavailableException.cs (1)</div>
<a href="../CredentialUnavailableException.cs.html#33"><b>33</b>: <i>base</i>(message, innerException)</a>
</div>
</div>
</body></html>